packages/venia-ui/lib/RootComponents/Category/categoryContent.js

```diff
@@ -41,26 +41,28 @@ const CategoryContent = props => {
 
     const classes = mergeClasses(defaultClasses, props.classes);
 
-    const header = filters ? (
-        <Fragment>
-            <div className={classes.headerButtons}>
-                <button
-                    className={classes.filterButton}
-                    onClick={handleOpenFilters}
-                    onFocus={handleLoadFilters}
-                    onMouseOver={handleLoadFilters}
-                    type="button"
-                >
-                    {'Filter'}
-                </button>
-                <ProductSort sortProps={sortProps} />
-            </div>
+    const maybeFilterButtons = filters ? (
+        <button
+            className={classes.filterButton}
+            onClick={handleOpenFilters}
+            onFocus={handleLoadFilters}
+            onMouseOver={handleLoadFilters}
+            type="button"
+        >
+            {'Filter'}
+        </button>
+    ) : null;
+
+    const maybeSortButton =
+        totalPagesFromData !== 0 ? <ProductSort sortProps={sortProps} /> : null;
```

Comment:
This prop can also be `null`, so lets just do a truthy check:

```suggestion
        totalPagesFromData ? <ProductSort sortProps={sortProps} /> : null;
```